src/mlpack/methods/ann/layer/concat.hpp

```diff
@@ -53,6 +53,19 @@ class Concat
   Concat(const bool model = false,
          const bool run = true);
 
+  /**
+   * Create the Concat object using the specified parameters.
+   *
+   * @param inputSize A vector denoting input size of each layer added.
+   * @param axis Concat axis.
+   * @param model Expose all network modules.
+   * @param run Call the Forward/Backward method before the output is merged.
+   */
+  Concat(arma::Row<arma::sword> inputSize,
+         const int axis = -1,
```

Comment:
What about removing the default value, in this case a user is forced to set the value, which is currently done by checking against ``-1``.

Reply:
Yes, that would allow me to change to `size_t` instead of `int` or `uword`. However, that would make things difficult for users. I will change it.

Reply:
Hm, the user has to pass the axis value anyway, so not sure I see how this makes things more difficult.